katas/args/alejandra/Schema.js

```diff
@@ -0,0 +1,60 @@
+export class Schema {
+    constructor() {
+        this.numberOfFlags = 3;
+        this.schemaArgs = [
+            {
+                flag: 'l',
+                name: 'logging',
+                type: 'boolean',
+                defaultValue: false,
+            },
+            {
+                flag: 'p',
+                name: 'port',
+                type: 'number',
+                defaultValue: 0,
+            },
+            {
+                flag: 'd',
+                name: 'directory',
+                type: 'string',
+                defaultValue: '',
+            },
+        ];
+    }
+
+    getSchemaArgsList() {
+        return this.schemaArgs;
+    }
+
+    findArg(flagToValidate) {
+        const args = this.schemaArgs;
+        const resultArg = args.find((arg) => arg.flag === flagToValidate);
+        return resultArg;
+    }
+
+    isAValidArg(flagToValidate, value) {
+        const foundedArg = this.findArg(flagToValidate);
+        if (!foundedArg) {
+            throw Error(
```

Comment:
Throwing an error could be an option, but in this case, since your function is called `isAValidArg`, and you return `true` in case the arg is valid, you can return `false` for the other cases since the value you are expecting to be returned by this function is a boolean.

Throwing errors would be more appropriate for handling unexpected situations that could happen in your code (exceptions, errors), but for a `isAValidArg` function, having an invalid value is expected, and for that, you return `false`, because the purpose of your function is to tell if an arg is valid or not